common/lib/api-client/models/index.js

```diff
@@ -34,6 +35,7 @@ module.exports = {
   allocation_complex_case: allocationComplexCase,
   assessment_question: assessmentQuestion,
   cancel,
+  category,
```

Comment:
This might be a little too late to determine if the API side has already been built but `category` feels a little ambiguous. I wonder if it would be better named as `prison_category` or `prisoner_category` (I can't remember which the Prison API uses..)?

Reply:
Will see what can be done

Reply:
Apparently the use of `category` is intentional as locations have also been rejigged to have them.